packages/eslint-plugin/src/configs/react-config.js

```diff
@@ -30,7 +29,6 @@ module.exports = {
     '@griffel/no-shorthands': 'error',
     '@griffel/pseudo-element-naming': 'error',
     '@griffel/styles-file': 'error',
-    'no-restricted-globals': restrictedGlobals,
```

Comment:
all these are not used anymore. for the new rule we use our `__internal__` pattern for workspace lint rules. see packages/eslint-plugin/src/internal.js